packages/hooks/src/__tests__/useIsSystemMaintenance.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { StoreProvider, TStores, mockStore } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useIsSystemMaintenance from '../useIsSystemMaintenance';
+
+describe('useIsSystemMaintenance', () => {
```

Comment:
@rostislav-deriv Let's follow the same conversation as `useCashierLocked.spec.tsx`.

Reply:
Improved the code in `useIsSystemMaintenance.spec.tsx`